src/js/utils/JSONUtil.js

```diff
@@ -0,0 +1,333 @@
+/**
+ * Tokenizer regular expressions
+ */
+const TOKENIZER_TOKENS = {
+  'comma': /,/g,
+  'end-label': /:/g,
+  'begin-object': /\{/g,
+  'end-object': /\}/g,
+  'begin-array': /\[/g,
+  'end-array': /\]/g,
+  'string': /"(\\["\\/bfnrtu"]|[^"\\"])*"/g,
```

Comment:
To be honest that's a bit of an overkill, but I think I found it as a 'suggested way of parsing escaped strings' on the collective knowledge of the internets.